docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.10-bullseye
+
+# install zbar dependencyy
+RUN apt-get -qq update
+RUN apt-get -y -qq install zbar-tools
+
+# temp add/copy seedsigner local repo to do pip3 install
+ADD . /seedsigner
+WORKDIR /seedsigner
+
+#COPY ../requirements.txt /requirements.txt
+#COPY ../tests/requirements.txt /tests-requirements.txt
+
+RUN pip3 install -r requirements.txt
+RUN pip3 install -r tests/requirements.txt
+RUN pip3 install -e .
```

Comment:
Could avoid the `ADD . /seedsigner` and invoke the `pip3 install -e .` in a bashrc file that we copy over (it could either check for "seedsigner" in `pip freeze` or just run the `install` on every login).

I tend to set a bashrc anyway that has terminal coloring and whatnot:
`COPY docker/bashrc /root/.bashrc`

```bash
case "$TERM" in
    xterm-color|*-256color) color_prompt=yes;;
esac

# uncomment for a colored prompt, if the terminal has the capability; turned
# off by default to not distract the user: the focus in a terminal window
# should be on the output of commands, not on the prompt
force_color_prompt=yes

if [ -n "$force_color_prompt" ]; then
    if [ -x /usr/bin/tput ] && tput setaf 1 >&/dev/null; then
	# We have color support; assume it's compliant with Ecma-48
	# (ISO/IEC-6429). (Lack of such support is extremely rare, and such
	# a case would tend to support setf rather than setaf.)
	color_prompt=yes
    else
	color_prompt=
    fi
fi

if [ "$color_prompt" = yes ]; then
    PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w \$\[\033[00m\] '
else
    PS1='${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '
fi
unset color_prompt force_color_prompt

# If this is an xterm set the title to user@host:dir
case "$TERM" in
xterm*|rxvt*)
    PS1="\[\e]0;${debian_chroot:+($debian_chroot)}\u@\h: \w\a\]$PS1"
    ;;
*)
    ;;
esac

# enable color support of ls and also add handy aliases
if [ -x /usr/bin/dircolors ]; then
    test -r ~/.dircolors && eval "$(dircolors -b ~/.dircolors)" || eval "$(dircolors -b)"
    alias ls='ls --color=auto'
    #alias dir='dir --color=auto'
    #alias vdir='vdir --color=auto'

    alias grep='grep --color=auto'
    alias fgrep='fgrep --color=auto'
    alias egrep='egrep --color=auto'
fi

cd /seedsigner
pip install -e .
```